Releasenotes.md

```diff
@@ -1,6 +1,68 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is hergebruik gemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
+  - De default-waarde voor de Accept-Crs headerparameter is verwijderd.
+  - Uit alle componentnamen zijn de underscores verwijderd.
+
+
+- KadastraalOnroerendeZaak :
+  - De default-waarde voor de Accept-Crs headerparameter is verwijderd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}
+  - PrivaatrechtelijkeBeperkingen, hypotheken en beslagen zijn toegevoegd aan de _embedded en _links. De bijbehorende identificaties zijn opgenomen in de KadastraalOnroerendeZaak.
+  - Bij LocatieKadastraalObject wordt nu het adres meegeleverd waar dat voorheen alleen een verwijzing met een nummeraanduidingIdentificatie was.
```

Comment:
Volgens mij volstaat één van deze:
`- LocatieKadastraalObject levert nu ook het adres.`
`- LocatieKadastraalObject is uitgebreid met het adres.`
`- LocatieKadastraalObject levert naast nummeraanduidingIdentificatie nu ook het adres.`